server/nodejs/package.json

```diff
@@ -6,10 +6,8 @@
     "start": "node .",
     "startd": "pm2 start node . $CONFIG --name 'sys-api-server'",
     "test": "mocha **/test/*.test.js --reporter spec --timeout 10000 --colors --exit",
-    "isolated": "APIRUNMODE=isolated node .",
-    "integrated": "APIRUNMODE=integrated node .",
-    "windows-isolated": "set APIRUNMODE=isolated&& node .",
-    "windows-integrated": "set APIRUNMODE=integrated&& node .",
+    "isolated": "cross-env APIRUNMODE=isolated node .",
```

Comment:
Nice stuff.  Needs to be verified by one of the Windows crew (@ShahnawazAli93 , @quangogster , and/or @oshchepkov ) for runnability but I'm glad to see cross-env here.

Reply:
Cool, will try this updated script and let you know.